### PR TITLE
feat: created routes to upload image to ocr

### PR DESCRIPTION
**Pull request by VisheshBansal**

### CODE REVIEW

**Comment by VisheshBansal** [comment]
@SaurusXI Pls review

File: tableDetection.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    for c in cnts:
+        cv2.drawContours(image, [c], -1, (36, 255, 12), 2)
+
+    # Detect vertical lines
+    vertical_kernel = cv2.getStructuringElement(cv2.MORPH_RECT, (1, 10))
+    detect_vertical = cv2.morphologyEx(thresh, cv2.MORPH_OPEN, vertical_kernel, iterations=2)
+    cnts = cv2.findContours(detect_vertical, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
+    cnts = cnts[0] if len(cnts) == 2 else cnts[1]
+    for c in cnts:
+        cv2.drawContours(image, [c], -1, (36, 255, 12), 2)
```
Comment (SaurusXI) [inline tableDetection.py:102]: Put redundant code from here in a separate function, call each time for drawing horizontal/vertical contours

File: main.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+'''Import necessary modules'''
+import cv2
+import numpy as np
+import uvicorn
+from fastapi import FastAPI, File, UploadFile
```
Comment (SaurusXI) [inline main.py:5]: Sort imports

**Review by SaurusXI — CHANGES_REQUESTED** [review]

**Review by SaurusXI — APPROVED** [review]

**Comment by SaurusXI** [comment]
@VisheshBansal feedback on commit messages - if you're making a commit to fix linting issues, don't make a separate commit for it as it doesn't signify a new/different state of your project. Instead, use rebasing to make linting fixes in the same commit.
More specifically, all 5 of your latest commits should have been squashed into a single commit.

**Comment by VisheshBansal** [comment]
> @VisheshBansal feedback on commit messages - if you're making a commit to fix linting issues, don't make a separate commit for it as it doesn't signify a new/different state of your project. Instead, use rebasing to make linting fixes in the same commit.
> More specifically, all 5 of your latest commits should have been squashed into a single commit.

Sure will keep it in mind. Thanks alot!

**Pull request merged**